presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewPlanValidator.java

```diff
@@ -32,15 +41,36 @@
 {
     private final Node query;
 
+    private final boolean isValidationOnly;
```

Comment:
I think the validation is the default functionality for this class. Should we call this flag as analyzeColumnMapping?

Reply:
Sure.